### DISCUSSION TITLE
Having trouble with form steps saving

### DISCUSSION THREAD

**Discussion by bdruth** [Budibase/budibase#11694] [Help]
Good evening,

n00b here, having a problem with getting a simple setup to work, following the docs for form steps ... also found https://github.com/Budibase/budibase/discussions/10965 and couldn't really get that working, either.

What's happening is that when I have a Save button, which does a Save Row action and then navigation to home screen, I just stay on the form step screen, with nothing displayed in the step area. Data isn't saved, and we don't navigate to the home screen.

I've tried nesting the Form under a data provider, that didn't change the behavior. If I do either a new screen -> Table, using the internal DS and built in create side bar, that works; or I use a Form Block with the data provider, that works ... but, neither of those options give me form steps which is what I was looking for.

Not sure what other debugging I can provide, it's all a very simple setup at this point - table has 4 rich text fields, form currently only has two steps, to enter the first two rich text fields. Using the docker image, if that matters?

Thanks in advance!

**Comment by melohagan** [comment]
Hey @bdruth 

Could you share some screenshots of your button actions, and your component tree please?

**Reply by bdruth** [reply]
<img width="259" alt="image" src="https://github.com/Budibase/budibase/assets/444571/0b5d8d1f-8a9e-4547-aa65-83565519cefb">

back button (doesn't work, fyi)
<img width="1006" alt="image" src="https://github.com/Budibase/budibase/assets/444571/c405d5dc-c8da-46fd-8392-c1861fa8ce3d">

next button
<img width="1000" alt="image" src="https://github.com/Budibase/budibase/assets/444571/e8f5a8ee-84b9-4b03-97f9-2f32b5b6df92">

save button
<img width="1002" alt="image" src="https://github.com/Budibase/budibase/assets/444571/79859c3e-ce36-41d4-8cc7-dda41ad7b301">

<img width="999" alt="image" src="https://github.com/Budibase/budibase/assets/444571/594281fd-9bdf-42a8-a41f-0853fc0836f6">

<img width="1009" alt="image" src="https://github.com/Budibase/budibase/assets/444571/dcd4b6ce-07c9-488d-ba3c-4051a43a4751">

**Reply by melohagan** [reply]
Thanks for the screenshots @bdruth 

I attempted to reproduce, but I was not able to see this:

> What's happening is that when I have a Save button, which does a Save Row action and then navigation to home screen, I just stay on the form step screen, with nothing displayed in the step area. Data isn't saved, and we don't navigate to the home screen.

Are there any errors in the console (Fn+F12 on Chrome) ?

May need an app export to explore further if you can email a download link to: support@budibase.com
